server/sonar-server/src/test/java/org/sonar/server/component/db/ComponentDaoTest.java

```diff
@@ -84,9 +88,9 @@ public void get_by_uuid() {
 
   @Test
   public void get_by_uuid_on_technical_project_copy() {
-    setupData("shared");
+    db.prepareDbUnit(getClass(), "shared.xml");
```

Comment:
since `ComponentDao` exposes quite a lot of methods which can be semantically grouped, a good refactoring could be to split the class in smaller ones or at least split the unit test. In both cases, it will then be possible to move the `db.prepareUnit` call to `@Before` method